lib/setup-lib.bash

```diff
@@ -140,9 +140,9 @@ function direnv_shell_integration() {
     *bash*)
       rcfile="$HOME/.bashrc"
       # shellcheck disable=SC2016
-      asdf_direnv_rcfile_expr='${XDG_CONFIG_HOME:-$HOME/.config}/asdf-direnv/bashrc'
+      asdf_direnv_rcfile_expr='"${XDG_CONFIG_HOME:-$HOME/.config}/asdf-direnv/bashrc"'
```

Comment:
Can this be `"` instead of `'"` ? 
Same on line 155.

Reply:
If we only use double-quotes, the value will be expanded. So instead of this getting written to the `.bashrc`:

```
source "${XDG_CONFIG_HOME:-$HOME/.config}/asdf-direnv/bashrc"
```

instead, on my system, it would write

```
source "/Users/chris/.config/asdf-direnv/bashrc"
```

My preference is to have the former, which is more portable and more consistent between users.